src/tests/ftest/launch.py

```diff
@@ -272,11 +272,31 @@ def get_output(cmd, check=True):
         cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
     stdout, _ = process.communicate()
     retcode = process.poll()
-    if check and retcode:
-        print(
+    if retcode:
+        raise RuntimeError(
             "Error executing '{}':\n\tOutput:\n{}".format(
                 " ".join(cmd), stdout))
-        exit(1)
+    return stdout
+
+def get_output(cmd, check=True):
+    """Get the output of given command executed on this host.
+
+    Args:
+        cmd (list): command from which to obtain the output
+        check (bool, optional): whether to emit an error and exit the
+            program if the exit status of the command is non-zero. Defaults
+            to True.
+
+    Returns:
+        str: command output
+    """
+    try:
+        stdout = run_command(cmd)
+    except RuntimeError as error:
+        if check:
+            print(error)
+            exit(1) 
+        stdout = str(error)
```

Comment:
(pylint-redefined-variable-type) Redefinition of stdout type from bytes to str

Reply:
Running linting with Py3 on Py2 code.